CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 * [CHANGE] Config flag category overrides can be set dynamically at runtime. #1934
 * [CHANGE] Ingester: deprecated `-ingester.ring.join-after`. Mimir now behaves as this setting is always set to 0s. This configuration option will be removed in Mimir 2.4.0. #1965
 * [CHANGE] Blocks uploaded by ingester no longer contain `__org_id__` label. Compactor now ignores this label and will compact blocks with and without this label together. `mimirconvert` tool will remove the label from blocks as "unknown" label. #1972
+* [CHANGE] Ruler: report failed evaluation metric for any 5xx status code returned by the query-frontend when remote operational mode is enabled. #2053
```

Comment:
I think this is a BUGFIX, not a breaking CHANGE.

Reply:
makes sense! 👍

Reply:
done @ ec1ca9fea0295dbc1c4c2bc4efc73d906b9f963e